debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresChangeRecordEmitter.java

```diff
@@ -292,6 +292,11 @@ private boolean schemaChanged(List<ReplicationMessage.Column> columns, Table tab
                         return true;
                     }
                 }
+                PostgresType pgType = message.getType();
+                if (pgType.isEnumType() && !pgType.getEnumValues().contains(message.asString())) {
```

Comment:
Where's `asMessage()` coming from?

Reply:
@bradengroom, could you shed any light on this one? Thx!

Reply:
You're referring to `message.asString()`, right? I haven't had time to look at this in a while, but I think that piece needed to be fixed up. `message.asString()` needs to be whatever the current enum value is. I assume there's a way to get that out of `message`?

Reply:
Yes, that's the method I was referring to. This is causing a compile error atm.